src/fonts/OTF/TeX/xbbold.mf

```diff
@@ -0,0 +1,182 @@
+%% filename: xbbold.mf
```

Comment:
How does this file get used?

Reply:
@kevinbarabash MetaFont first searches `cwd` for the source file, so instead of the one included in the LaTeX distribution, this file will be used in tracing `msbm10` and others.